pkgs/by-name/ka/kanboard/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nixosTests,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation (attrs: {
```

```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
```

Bit of a nitpick, but `finalAttrs` is the pseudo-standard name for this lambda